crs-setup.conf.example

```diff
@@ -177,6 +181,30 @@ SecDefaultAction "phase:2,log,auditlog,pass"
 #   setvar:tx.paranoia_level=1"
 
 
+#
+# -- [[ Enforce Body Processor URLENCODED ]] -----------------------------------
+#
+# ModSecurity selects the body processor based on the Content-Type request
+# header. But clients are not always setting the Content-Type header for their
+# request body payloads. This will leave ModSecurity with limited vision into
+# the payload.  The variable tx.enforce_bodyproc_urlencoded lets you force the
+# URLENCODED body processor in these situations. This is off by default, as it
+# implies a change of the behaviour of ModSecurity beyond CRS (the body
+# processor applies to all rules, not only CRS) and because it may lead to
+# false positives already on paranoia level 1. However, in a high security
```

Comment:
I would mention that this is not only for High security but it also prevents a big security bypass when a client purposely omit the Content-Type header to prevent the request to be parsed by any body processor which will result in many of the variables used by the CRS to perform most of its inspections to not be created.

Reply:
We have a rule for that case - 920340 - but it is only on level notice / 2 points. Would it make sense to raise said rule to 5 points and automatically lead to a block in the default installation?

Reply:
Hi @dune73, I think there are many clients that send little headers, like mobiles and web sites that use javascript to do requests on the background. That change would result in an undesired update nightmare, personally I would prefer to first push the message in the comments and maybe rephrase the message on 920340 to highlight this fact (for those that do not read comments) and consider a 920340 sibling in higher PL to increase the score in PL3 maybe.

Reply:
I have reworded crs-setup.conf.example after your suggestion and also added a notice to 920340.